aggregate_spatial_window.json

```diff
@@ -0,0 +1,98 @@
+{
+    "id": "aggregate_spatial_window",
+    "summary": "Zonal statistics for rectangular windows",
+    "description": "Aggregates statistics over the horizontal spatial dimensions (axes `x` and `y`) of the data cube.\n\nThe pixel grid for the axes `x` and `y` is divided into non-overlapping windows with the size specified in the parameter `size`. If the number of values for the axes `x` and `y` is not a multiple of the window size, the behaviour specified in `boundary` parameter is applied.\nFor each of these windows, the reducer process computes the result.",
+    "categories": [
+        "cubes",
+        "aggregate & resample"
+    ],
+    "parameters": [
+        {
+            "name": "data",
+            "description": "A raster data cube.",
```

Comment:
Clarification needed, see https://github.com/Open-EO/openeo-processes/pull/192#discussion_r490087458

> Does it have constraints? It seems like the input raster data cube must have only two dimensions? x and y...

Proposal from @clausmichele:
> A raster data cube with two spatial dimensions and arbitrary number of bands and times. If the datacube has multiple bands the process will be applied to all of them. Same for a timeseries.

Reply:
@clausmichele Do you think the following wording would work?

> A raster data cube with exactly two horizontal spatial dimensions and an arbitrary number of additional dimensions. The process is applied to all additional dimensions individually.

Reply:
@m-mohr yes the description is fine!